inference-engine/src/vpu/myriad_plugin/myriad_executor.cpp

```diff
@@ -266,15 +266,15 @@ DevicePtr MyriadExecutor::openDevice(std::vector<DevicePtr>& devicePool,
                 return device->isBooted() && device->isNotFull()
                        && device->isSuitableForConfig(config);
             });
-
         // Return mock device. If try infer with it, exception will be thrown
-        if (availableDevices.empty() && config.platform() != NC_ANY_PLATFORM) {
+        if (availableDevices.empty()) {
+            if (config.platform() == NC_ANY_PLATFORM) {
+                _log->warning("No booted device");
```

Comment:
Let's tell a user that myriad plugin is going to compile model but inference will be unavailable